qutip/solver/stochastic.py

```diff
@@ -0,0 +1,754 @@
+        spawn seeds for each trajectory or a list of seeds, one for each
+        trajectory. Seeds are saved in the result and they can be reused with::
+
+            seeds=prev_result.seeds
+
+        When using a parallel map, the trajectories can be re-ordered.
+
+    target_tol : {float, tuple, list}, optional
+        Target tolerance of the evolution. The evolution will compute
+        trajectories until the error on the expectation values is lower than
+        this tolerance. The maximum number of trajectories employed is
+        given by ``ntraj``. The error is computed using jackknife resampling.
+        ``target_tol`` can be an absolute tolerance or a pair of absolute and
+        relative tolerance, in that order. Lastly, it can be a list of pairs of
+        ``(atol, rtol)`` for each e_ops.
+
+    timeout : float [optional]
+        Maximum time for the evolution in second. When reached, no more
+        trajectories will be computed. Overwrite the option of the same name.
+
+    options : None / dict
+        Dictionary of options for the solver.
+
+        - store_final_state : bool, [False]
+          Whether or not to store the final state of the evolution in the
+          result class.
+        - store_states : bool, None, [None]
+          Whether or not to store the state vectors or density matrices.
+          On `None` the states will be saved if no expectation operators are
+          given.
+        - store_measurement: bool, [False]
+          Whether to store the measurement and wiener process for each
+          trajectories.
+        - keep_runs_results : bool, [False]
+          Whether to store results from all trajectories or just store the
+          averages.
+        - normalize_output : bool, [False]
+          Normalize output state to hide ODE numerical errors.
+        - progress_bar : str {'text', 'enhanced', 'tqdm', ''}, ["text"]
+          How to present the solver progress.
+          'tqdm' uses the python module of the same name and raise an error
+          if not installed. Empty string or False will disable the bar.
+        - progress_kwargs : dict, [{"chunk_size": 10}]
+          kwargs to pass to the progress_bar. Qutip's bars use `chunk_size`.
+        - method : str, ["rouchon"]
+          Which stochastic differential equation integration method to use.
+          Main ones are {"euler", "rouchon", "platen", "taylor1.5_imp"}
+        - map : str {"serial", "parallel", "loky"}, ["serial"]
+          How to run the trajectories. "parallel" uses concurent module to run
+          in parallel while "loky" use the module of the same name to do so.
+        - job_timeout : NoneType, int, [None]
+          Maximum time to compute one trajectory.
+        - num_cpus : NoneType, int, [None]
+          Number of cpus to use when running in parallel. ``None`` detect the
+          number of available cpus.
+        - dt : float [0.001]
```

Comment:
It seems the default value for `dt` is not always `0.001` but it is set differently in a few places. Is the default integrator specific? Perhaps this should be left in the integrator options? Not sure.